SAMtags.tex

```diff
@@ -246,13 +247,17 @@ \subsection{Barcodes}
 \end{description}
 
 \subsection{Original data}
-
 \begin{description}
```

Comment:
This blank line is consistent with the formatting elsewhere.  No need to make it inconsistent...